docs-new/docs/graphql/core/api-reference/index.mdx

```diff
@@ -0,0 +1,111 @@
+---
+description: Hasura API reference
+keywords:
+  - hasura
+  - docs
+  - API
+  - API reference
+slug: index
+---
+
+# API Reference
+
+## Available APIs
+
+| API                            | Endpoint                                         | Access           |
+|--------------------------------|--------------------------------------------------|------------------|
+| GraphQL                        | [/v1/graphql](#graphql-api)                      | Permission rules |
+| Relay                          | [/v1beta1/relay](#relay-api)                     | Permission rules |
+| Legacy GraphQL                 | [/v1alpha1/graphql](#graphql-api)                | Permission rules |
+| Schema *(> v2.0)*              | [/v2/query](#schema-api)                         | Admin only       |
+| Metadata *(> v2.0)*            | [/v1/metadata](#metadata-api)                    | Admin only       |
+| Schema/Metadata *(deprecated)* | [/v1/query](#schema--metadata-api-deprecated)    | Admin only       |
+| Restified GQL                  | [/api/rest](#restified-graphql-api)              | GQL REST Routes  |
+| Version                        | [/v1/version](#version-api)                      | Public           |
+| Health                         | [/healthz](#health-check-api)                    | Public           |
+| PG Dump                        | [/v1alpha1/pg_dump](#pg_dump-api)                | Admin only       |
+| Config                         | [/v1alpha1/config](#config-api)                  | Admin only       |
+| Explain                        | [/v1/graphql/explain](#explain-api)              | Admin only       |
```

Comment:
Though all these sub section links work, these rely on the auto generated ids and are prone to break if the section title is changed.

This table requires only one change - `schema--metadata-api-deprecated` which is from autogenerated and has `--` due to a `/` slash in the header (`Schema / metadata API (Deprecated)`). Please replace it to `schema-metadata-api` and add custom header id for the respective section.

Similaryly, please add respective custom header ids for all the sections in this file.

Reply:
resolved